docs/logicapps-ai-course/02_build_conversational_agents/05-add-user-context-to-tools.md

```diff
@@ -48,56 +48,42 @@ Many solutions use mixed authorization methods. For example, a solution might re
-    - [Configure your App Service or Azure Functions app to use Microsoft Entra sign-in](https://learn.microsoft.com/azure/app-service/configure-authentication-provider-aad)
-
-  - The conversational agent workflow requires a connector operation that works with a Microsoft service or system. The connection must also support delegated, per-user connections, for example, Microsoft 365, SharePoint, or Microsoft Graph. If required, the connection might also need tenant administrator consent.
-
-    > [!TIP]
-    > 
-    > Make sure to keep tool descriptions concise and include guidance about the data that they access,
-    > for example, such a description might say "Gets the signed-in user's next five calendar events."
-
-- The chat client interface integrated with conversational agent workflows.
+  - The conversational agent workflow requires a connector operation that works with a Microsoft service or system. The connection must also support delegated, per-user connections, for example, Microsoft 365, SharePoint, or Microsoft Graph.
 
-  This chat client handles token delegation for per-user connections. If you want to use a custom client, the client must pass the user token to the agent.
+- The chat client interface integrated with conversational agent workflows. For this module, we will use the integrated chat client provided by the logic app. You will learn how to use a custom client in [Module 09](./09-deploy-agents-clients.md).
 
-  > [!NOTE]
-  >
-  > If you are building a custom client that can't pass a user token yet, complete the concepts with the app-only connections.
-  > In a later module, you integrate a custom client that supplies the user's access token to the agent.
+- Preferably two test users or user accounts for conversing through chat with different permissions in target connection.
 
-- At least two test users for conversing through chat with different permissions in your tenant.
+  For example, one user account might have access to a mailbox or website, while the other user account doesn't have access.
 
-  For example, one user might have access to a mailbox or website, while the other user doesn't have access.
+## Recommended - Set up Easy Auth (App Service authentication and authorization) on your logic app
 
-## Part 1 - Set up Easy Auth on your logic app
-
-Your Standard logic app resource requires that you set up Easy Auth so that the delegated identity can flow. At runtime, the chat client integrated with Azure Logic Apps prompts each first-time user to authenticate and uses their identity afterwards.
+For production scenarios, including chat clients outside of the Azure Portal, we recommend setting up Easy Auth to securely handle authentication and authorization of user credentials.
 
 1. In the [Azure portal](https://portal.azure.com), open your Standard logic app resource.
```

Comment:
Manual numbering means you have to renumber the steps should you change them enough to result in changing the numbering. The "1.", "1.", "1." pattern automatically renumbers the steps in the correct order. The guidelines that I posted explained this.